UITests/Global.swift

```diff
@@ -238,7 +238,7 @@ class BrowserUtils {
 
         let goPrivateModeBtn = grey_allOf([grey_accessibilityID("TabTrayController.maskButton"), grey_accessibilityValue("Off")])
         let goNormalModeBtn = grey_allOf([grey_accessibilityID("TabTrayController.maskButton"), grey_accessibilityValue("On")])
-        let closeAllBtn = grey_allOf([grey_accessibilityLabel("Close All Tabs"), grey_kindOfClass(NSClassFromString("_UIAlertControllerActionView")!)])
+        let closeAllBtn = grey_allOf([grey_accessibilityLabel("TabTrayController.deleteButton.closeAll"), grey_kindOfClass(NSClassFromString("_UIAlertControllerActionView")!)])
```

Comment:
Since the ID has been added, this needs to be changed to use `grey_accessibilityID` insteado of `grey_accessibilityLabel`